source/nodejs/adaptivecards-templating/src/shared.ts

```diff
@@ -2,4 +2,6 @@
 // Licensed under the MIT License.
 export class GlobalSettings {
     static undefinedExpressionValueSubstitutionString?: string = undefined;
+    static expressionStartMarker = "${";
```

Comment:
>${"; [](start = 36, length = 4)

This would need to be updated in the documentation/samples ? If so can you do me a favor, update your issue with a tasklist style todo in there (- [ ] is the syntax) and capture that in there for starters (unsure yet if we need a completely separate issue for that for now)?

Reply:
None of the flags in GlobalSettings are used at this time. If these end up staying then I'll make sure the capabilities are documented.